tests/unit_tests/test_wave_06.py

```diff
@@ -110,16 +107,19 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
Be careful using `==` for asserts on lists! It requires that the items be in exactly the same order to be true. It would be safer to use `in` like the other tests in this file.